packages/image_picker/image_picker_macos/example/macos/RunnerTests/ImageCompressTests.swift

```diff
@@ -0,0 +1,37 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import XCTest
+
+@testable import image_picker_macos
+
+final class ImageCompressTests: XCTestCase {
+
+  private func createTestImage(size: NSSize) -> NSImage {
+    let image = NSImage(size: size)
+    image.lockFocus()
+    NSColor.white.set()
+    NSBezierPath(rect: NSRect(origin: .zero, size: size)).fill()
+    image.unlockFocus()
+    return image
+  }
```

Comment:
This is duplicated in both `ImageCompressTests` and `ImageResizeTests`, should I extract it?